anaconda.spec.in

```diff
@@ -425,12 +425,14 @@ rm -rf \
 %exclude %{python3_sitearch}/pyanaconda/ui/gui/*
 %exclude %{python3_sitearch}/pyanaconda/ui/tui/*
 %{_bindir}/anaconda-cleanup
+# Make sure to not overwrite user configurtion files by using '(noreplace)'
```

Comment:
s/configurtion/configuration

Reply:
I guess the comment is correct because this noreplace does that. however, I dont see a scenario where anaconda would run on a existing system where these files exist.
I would even remove the comment, or comment that noreplace is there to silence rpmlint.

Reply:
Done